autosklearn/estimators.py

```diff
@@ -358,6 +359,54 @@ def fit(self, **kwargs):
 
         return self
 
+    def fit_pipeline(
+        self,
```

Comment:
Could you please give the full signature here? I think that'll be helpfull as this is the user facing function.